build.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash -e
+# =====================================================================
+# Build script for Freifunk Fulda firmware running on Jenkinis CI
+#
+# Source: https://github.com/freifunk-fulda
+# Contact: fffd-noc@lists.open-mail.net
+# Web: https://fulda.freifunk.net
+#
+# Credits:
+#   - Freifunk Darmstadt for your great support
+# =====================================================================
+
+# Default make options
+MAKEOPTS="-j 9 V=s"
+
+# Help function used in error messages and -h option
+usage() {
+  echo ""
+  echo "Build script for Freifunk-Fulda gluon firmware."
+  echo ""
+  echo "-c: Build command, [ update | download | build | sign | upload ]"
+  echo "-b: String for git branch, e.g. development"
+  echo "-d: Enable bash debug output"
+  echo "-h: Show this help"
+  echo "-m: Optional setting for make options. Default is \"-j 4\""
+  echo "-n: String for build number, e.g. b48"
+  echo "-w: Path for workspace, e.g. current work directoy"
+}
+
+# Evaluate arguments for build script.
+while getopts b:c:dhm:n:w: flag; do
+  case ${flag} in
+    b)
+      GIT_BRANCH=${OPTARG};
+      ;;
+    c)
+      case ${OPTARG} in
+        update)
+          export COMMAND=update
+          ;;
+        download)
+          export COMMAND=download
+          ;;
+        build)
+          export COMMAND=build
+          ;;
+        sign)
+          export COMMAND=sign
+          ;;
+        upload)
+          export COMMAND=upload
+          ;;
```

Comment:
thos can be shortened to something like:

update)
download)
build)
...
    export COMMAND="${OPTARG}"
    ;;